### ISSUE TITLE
investigate issues with image assets

### ISSUE THREAD

**Issue by nylki** [nylki/aframe-fit-texture-component#2]
When testing there where errors thrown by _three.js_ when using a texture via aframes asset-management and fit-texture together. Direct URLs didn't yield any errors. Gonna take a look whats wrong there.

**Comment by B-Lemke** [comment]
Did you ever get this working? I'm still getting some issues with this where sometimes it'll work with assets, sometimes not.

**Comment by nylki** [comment]
@B-Lemke I don't think so, I am afraid. Unfortunately I also don't have much spare time currently to take a fresh look at it, either.
So, any PRs or ideas in this regards are very welcome! :)
